protos/core/Btfs.proto

```diff
@@ -0,0 +1,256 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "org.tron.protos"; //Specify the name of the package that generated the Java file
+option java_outer_classname = "Protocol"; //Specify the class name of the generated Java file
+
+
+package btfs;
+
+
+service guard {
```

Comment:
Before we finalize, please make sure the naming follows standard protobuf conventions: https://developers.google.com/protocol-buffers/docs/style

Reply:
I think there are some tools online to help with this

Reply:
done the change for the format using lint provide by weiyu